pkg/backend/display/object_diff.go

```diff
@@ -1093,9 +1093,9 @@ func (p *propertyPrinter) printCharacterDiff(diffs []diffmatchpatch.Diff) {
 	for _, d := range diffs {
 		switch d.Type {
 		case diffmatchpatch.DiffDelete:
-			p.withOp(deploy.OpDelete).write(escape(d.Text))
+			p.withOp(deploy.OpDelete).writeVerbatim(escape(d.Text))
```

Comment:
To make this easier to review, here's the difference between `write` and `writeVerbatim`:

https://github.com/pulumi/pulumi/blob/ff8c44f9f51a3197aca11a7632e7f7a5a431c3de/pkg/backend/display/object_diff.go#L548-L554

https://github.com/pulumi/pulumi/blob/ff8c44f9f51a3197aca11a7632e7f7a5a431c3de/pkg/backend/display/object_diff.go#L110-L116